### ISSUE TITLE
Can't import `tkinter` in Python 2.7

### ISSUE THREAD

**Issue by Marrin** [PythonCharmers/python-future#262]
I get an error when trying to import `tkinter`:

```
Traceback (most recent call last):
  File "forum4.py", line 2, in <module>
    from tkinter import *
  File "/usr/local/lib/python2.7/dist-packages/tkinter/__init__.py", line 6, in <module>
    from Tkinter import (_cnfmerge, _default_root, _flatten, _join, _setit,
ImportError: cannot import name _join
```

The import of `_join` was added as fix for #233 and also includes `_splitdict`, and `_stringify` which don't exist in my Python 2.7s `Tkinter` module.  The perils of implementation details I guess.

**Comment by DaveyBiggers** [comment]
I'm also having this problem on Ubuntu 14.04 and Debian 7 - _splitdict seems to have been introduced around Python 2.7.9; it doesn't exist in 2.7.6, for example, which is the default system python for Ubuntu 14.04.

**Comment by sebfio** [comment]
This error goes away after capitalizing tkinter's import. Change 'tkinter' to 'Tkinter' and it'll hopefully go away!

**Comment by Marrin** [comment]
@sebfio That defeats the point of using `future` to import the old Python 2 module that is renamed and changed in Python 3, doesn't it?

**Comment by mpurg** [comment]
I noticed the same issue as @DaveyBiggers with Ubuntu 14.04.5 LTS, Python 2.7.6, future 0.16.0
```
Traceback (most recent call last):
  File "file.py", line 42, in <module>
    import tkinter
  File "/home/asd/.virtualenvs/asd/local/lib/python2.7/site-packages/tkinter/__init__.py", line 6, in <module>
    from Tkinter import (_cnfmerge, _default_root, _flatten, _join, _setit,
ImportError: cannot import name _splitdict
```
I created a pull request with a simple fix.

**Comment by jmadler** [comment]
Resolved by #379
